### ISSUE TITLE
Investigate current shipping engine efficiency

### ISSUE THREAD

**Issue by nworbmot** [PyPSA/pypsa-eur#555]
To convert the shipping fuel demand from today's fuel oil to tomorrow's hydrogen, we need to know the average efficiency of shipping engines today.
As a first guess I've set 'shipping_average_efficiency' in config.yaml to 0.4.

**Comment by gincrement** [comment]
@euronion do you know if this is covered already somewhere?

**Comment by fneum** [comment]
No: https://github.com/PyPSA/pypsa-eur/blob/75a73ca515e5b6cfb174191ffabb52ae32d5785b/config/config.default.yaml#L799
